strelets/adapter/swarm_volumes.go

```diff
@@ -63,6 +63,14 @@ func (d *dockerSwarmOrchestrator) provisionVolume(ctx context.Context, volumeNam
 	}, nil
 }
 
+func (d *dockerSwarmOrchestrator) provisionNetworkVolume(ctx context.Context, volumeName string, target string, maxSizeInGb int) (mount.Mount, error) {
```

Comment:
this function is expendable. please remove it

Reply:
I would rather have two functions that clearly state what they intend to do